tests/unit-test.el

```diff
@@ -0,0 +1,62 @@
+(require 'paredit)
+(require 'clj-refactor)
+
+(describe "completable-for-cljr-slash?"
+  (it "Returns `t' for tokens that can represent alias-prefixed named things (symbols, keywords, metadata)"
+    (expect (completable-for-cljr-slash? nil) :to-be nil)
+    (dolist (prefix '(""
```

Comment:
What about `#'my-alias/private-var`? 

Think that's the only other form that I use regularly that isn't covered.

Reply:
Good call! backtick also 

(unrelated, I think some CIDER features don't parse #' or backtick either...)